cirq/ops/common_channels.py

```diff
@@ -187,3 +189,528 @@ def depolarize(p: float) -> DepolarizingChannel:
         ValueError if p is not a valid probability.
     """
     return DepolarizingChannel(p)
+
+
+class GeneralizedAmplitudeDampingChannel(raw_types.Gate):
+    """ The generalized amplitude damping channel."""
+
+    def __init__(self, p, gamma) -> None:
+
+        """
+        This channel models the effect of dissipation to the environment
+        at possibly non zero tempurature. Where gamma is the damping rate
+        and p is the probability of
+
+        This channel evovles a density matrix via
+            \rho -> M_0 \rho M_0^\dagger + M_1 \rho M_1^\dagger
+                  + M_2 \rho M_2^\dagger + M_3 \rho M_3^\dagger
+
+        With
+            M_0 = \sqrt{p} \begin{bmatrix} 1 & 0  \\
```

Comment:
Horizontally align the text like so:

```
\begin{bmatrix}
   1 & 0 \\
    0 & ...
```

so that it can be read by eye easily.